integration-test/1910-underground-water.py

```diff
@@ -0,0 +1,86 @@
+# -*- encoding: utf-8 -*-
+from shapely.wkt import loads as wkt_loads
```

Comment:
Nit: Since we're not actually loading any WKT then remove the import (instead we're relying on dsl)